webapp/app/routes/singleList/singleList.component.js

```diff
@@ -0,0 +1,144 @@
+import React, { PureComponent, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { ifElse, always } from 'ramda';
+import Typograpghy from 'material-ui/Typography';
+import IconButton from 'material-ui/IconButton';
+import List, { ListItem, ListItemSecondaryAction } from 'material-ui/List';
+import Checkbox from 'material-ui/Checkbox';
+import Divider from 'material-ui/Divider';
+import Button from 'material-ui/Button';
+import AddIcon from '@material-ui/icons/Add';
+import Zoom from 'material-ui/transitions/Zoom';
+import Collapse from 'material-ui/transitions/Collapse';
+import HomeIcon from '@material-ui/icons/Home';
+import KeyboardArrowRightIcon from '@material-ui/icons/KeyboardArrowRight';
+import EditIcon from '@material-ui/icons/Edit';
```

Comment:
All icons could be also taken via 'material-ui-icons' package

Reply:
material-ui-icons is deprecated

https://www.npmjs.com/package/material-ui-icons

Reply:
I didn't know that, thanks for info!